honeybee_radiance/dynamic/state.py

```diff
@@ -573,18 +573,20 @@ def scale(self, factor, origin=None):
         if self._dmtx_geometry:
             self._dmtx_geometry = self._dmtx_geometry.scale(factor, origin)
 
-    def vmtx_to_radiance(self, minimal=False):
+    def vmtx_to_radiance(self, modifier=white_glow, minimal=False):
         """Generate a RAD string representation of this state's vmtx.
 
         The resulting string lacks modifiers and only includes the vmtx_geometry.
 
         Args:
+            modifier: A modifier (only identifier) assigned to the vmtx_geometry.
+                Default: white_glow.
```

Comment:
It's unclear what (only identifier) means here. Clearly you want the modifier object and not the text of the identifier, right?

Reply:
That's correct. It should be fixed now.